src/game/game_player.c

```diff
@@ -26,11 +26,15 @@ void game_player_free(game_player *gp) {
 }
 
 void game_player_set_har(game_player *gp, object *har) {
-    gp->har = har;
+    if (har) {
```

Comment:
Looks strange that this supports a NULL har, and is a void function, so caller has no idea what happened here..?

Reply:
we null the HAR out when the fight ends, but we used to just store a pointer, now we store an object ID, so we need to check for the null pointer